CHANGELOG.rst

```diff
@@ -1,7 +1,7 @@
 Changelog for Pulse3D
 =====================
 
-0.29.0 (unreleased)
+0.29.0 (2023-01-23)
```

Comment:
0.29.0 is already published, the changelog just wasn't updated yet, but it was released on 2022-12-22. You'll need to make a new entry for 0.29.1